docs/tips-and-tricks.md

```diff
@@ -31,3 +31,12 @@ from stac_fastapi.extensions.core.context import ContextExtension
 ```
 
 and then edit the `api = StacApi(...` call to add `ContextExtension()` to the list given as the `extensions` parameter.
+
+## Set API title, description and version
+
+For the landing page, you can set the API title, description and version using environment variables.
+
+- `STAC_FASTAPI_VERSION` (string) is the version number of your API instance (this is not the STAC version).
+- `STAC FASTAPI_TITLE` (string) should be a self-explanatory title for your API.
+- `STAC FASTAPI_DESCRIPTION` (string) should be a good description for your API. It can contain CommonMark.
+- `STAC_FASTAPI_ID` (string) is a unique identifier for your API.
```

Comment:
@m-mohr what's the difference between `STAC_FASTAPI_VERSION` and `STAC_FASTAPI_ID` ?

Reply:
One sets the id field in the landing page, the other set's the version of STAC fastapi afaik.

Reply:
but why would you want those to be different?

Reply:
🤔 ok I think I understand, IMO `STAC_FASTAPI_ID` should be renamed `STAC_FASTAPI_LANDING_ID` to be a bit more verbose. I'm also bit curious how ofter people will want to set this via env variable 🤷

Reply:
Why should it be kept stac-fastapi? Having all APIs expose the same id seems like a bad practice.
Maybe the names should actually be:

- STAC_CATALOG_ID
- STAC_CATALOG_TITLE
- STAC_CATALOG_DESCRIPTION
- STAC_FASTAPI_INSTANCE_VERSION

?

Reply:
🤔 I'm not sure I quite get this. if you're talking about the env prefix `STAC_FASTAPI`, to me it makes sense because you want to setup something within the stac-fastapi application! This way you're trying not to conflict with other application. 

In term of variable, I think it's a big mess because we're using the variable to set information in the OpenAPI document and the Landing page which are both totally different 🤷

Reply:
Okay, fine for me. I wouldn't add the `_LANDING` myself because it is an implementation detail. Imaging OpenAPI 4 having an id field in the info section, then you would want to remove the LANDING_ again. But as long as there's a way to configure it, I'm fine with whatever is done ;-)